userguide/recommender/coreml-deployment.md

```diff
@@ -0,0 +1,69 @@
+# Deployment to Core ML
+
+***Note:*** *This functionality is available in the beta release of Turi Create 5.0. For a 
+full list of 5.0 features and installation instructions, see 
+[release notes](https://github.com/apple/turicreate/releases/tag/5.0b1)*
+
+***Note:*** *This functionality is only available on iOS 12+ and macOS 10.14+*
+
+The Turi Create Recommender Model is also available for use in your iOS/macOS
+apps as a custom model via exporting to Core ML. After creating your model in Turi
+Create, save it in Core ML format by calling `export_coreml` API as follows:
+
+```python
+# assume my_recommender is the trained Turi Create Recommender Model
+my_recommender.export_coreml("MyRecommender.mlmodel")
+```
+
+After you drag and drop the exported Core ML model in your iOS app, it will look something like this:
+![Core ML Model Screenshot in Xcode](xcode-shot.png)
+
+Before you can go ahead and use the recommender model in your app, you must
+drag and drop the relevant `.dylib` into your app. If you are writing an app 
+for a Mac or an iOS simulator, you would want `libRecommender-x86_64.dylib`. 
```

Comment:
Per my earlier comment. You didn't mention how to exactly do this step. You have to drag and drop it and then make sure its in the copy phases. Attach a screenshot of that.